test/GovernanceStakerOnBehalf.t.sol

```diff
@@ -1284,7 +1283,7 @@ contract ClaimRewardOnBehalf is GovernanceStakerRewardsTest {
 
     // A user deposits staking tokens
     (_depositAmount, _depositId) =
-      _boundMintAndStake(_depositor, _depositAmount, _delegatee, _claimer);
+      _boundMintAndStake(_depositor, _depositAmount, address(0x01), _claimer);
```

Comment:
I don't understand why this change was made?

Reply:
Good question, this is to avoid a stack too deep error